index.js

```diff
@@ -126,13 +126,9 @@ cm.Item({
 });
 
 function checkLocation(opts) {
```

Comment:
I just looked at the elements again and they're <video> which I think you said is not Vine support but HTML5 support? So maybe, for this PR, I can delete checkLocation and where it's called, push this up so at least Vine hrefs are supported, and then open a new issue for HTML5 support. What do you think?